src/ui/components/atoms/label-date/LabelDate.scss

```diff
@@ -0,0 +1,19 @@
+@import '../../../../globals/breakpoints';
+@import '../../../../globals/fonts';
+@import '../../../../globals/mixins';
+@import '../../../../globals/variables';
+
+.labeldate {
+  color: $ads-text-secondary;
+  font-family: Lato, Arial, Helvetica, sans-serif;
+  font-size: 12px;
```

Comment:
don't set a size from here use the globals 
$ads-font-size-second: 1.75rem;
$ads-font-size-quarter: 2.5rem;
$ads-font-size-fifth: 0.75rem;
$ads-font-size-sixth: 0.938rem;
$ads-font-weight-first: 400;
$ads-font-weight-second: 800;